packages/server/graphql/mutations/pushInvitation.ts

```diff
@@ -45,6 +46,16 @@ export default {
       if (!isAuthenticated(authToken)) return standardError(new Error('Not authenticated'))
 
       // VALIDATION
+      const team = await dataLoader.get('teams').load(teamId)
+      if (!team) {
+        return {error: {message: 'Invalid teamId'}}
+      }
+
+      const {orgId} = team
+      const approvalError = await getIsUserIdApprovedByOrg(viewerId, orgId, dataLoader)
```

Comment:
if someone goes to the meeting url when they're not on the team & they don't have an org-approved email address, then a push notification is **not** pushed to the other team members asking if they want to invite the stranger to their team.